.taskcluster.yml

```diff
@@ -153,8 +156,7 @@ tasks:
             - "-lcx"
             - "apt-get -qq update &&
               apt-get -qq install -y libhdf5-dev &&
-              git clone --quiet ${repository} &&
-              cd bugbug &&
```

Comment:
Nit: no need to move this to a single line

Reply:
@marco-c  and @suhaibmujahid Alright, will revert, thanks It's still failing code coverage. Can I add unnittest for [download_model.py ](https://github.com/mozilla/bugbug/blob/master/http_service/bugbug_http/download_models.py) file so as to bump the coverage? I'm thinking i should do it in another branch but it won't solve the problem as the branch lowers the code coverage.

Reply:
> It's still failing code coverage. 

It is ok.

> Can I add unnittest for [download_model.py ](https://github.com/mozilla/bugbug/blob/master/http_service/bugbug_http/download_models.py) file so as to bump the coverage? I'm thinking i should do it in another branch but it won't solve the problem as the branch lowers the code coverage.

Yes, you could file an issue for that and open a PR to resolve it.